v1.1/training/architecture-overview.md

```diff
@@ -6,8 +6,26 @@ sidebar_data: sidebar-data-training.json
 redirect_from: /training/architecture-overview.html
 ---
 
+Watch the rest of Alex Robinson's talk, where he explains the CockroachDB architecture and how it was built.
```

Comment:
Nit: I'd put this sentence inside the Video div.

Reply:
I've left it at the top but mentioned the slides as well.